pom.xml

```diff
@@ -31,7 +31,7 @@
   <artifactId>parent</artifactId>
   <packaging>pom</packaging>
   <!-- to change version: mvn -\-batch-mode release:update-versions -DdevelopmentVersion=3.6.0-SNAPSHOT -->
-  <version>3.7.0-SNAPSHOT</version>
+  <version>${revision}</version>
```

Comment:
In my opinion this won't work together with the maven release plugin.
We are using it in order to automate the release process.
Is it necessary for you ?

Reply:
No it's not necessary.

Reply:
I've updated the patch. Thanks.